src/ensembl/src/content/app/species-selector/containers/species-search-panel/SpeciesSearchPanel.scss

```diff
@@ -2,9 +2,10 @@
 
 .speciesSearchPanel {
   background-color: $ens-light-grey;
-  height: 270px;
+  height: 235px;
   padding: 65px 40px;
-  padding-left: 100px; // FIXME: should this be shared variable? it will be reused by PopularSpeciesPanel
+  padding-left: $global-padding-left;
+  box-shadow: 0 3px 5px $global-box-shadow;
```

Comment:
Should `$global-box-shadow` variable be also used in the Home screen scss [here](https://github.com/Ensembl/ensembl-client/blob/dev/src/ensembl/src/content/home/Home.scss#L15) then?

Reply:
Yes, I think so - I can do a tidy-up for next release.